pages.fr/common/z.md

```diff
@@ -1,6 +1,6 @@
 # z
 
-> Recherche les répertoires les plus utilisés et permet une navigation rapide à l'aide de chaine de caractères ou de regex.
+> Recherche les répertoires les plus utilisés et permet une navigation rapide à l'aide de chaine de caractères ou d'expressions régulières.
```

Comment:
This change is already happening in #5830. I'm using `de expression régulière`, but I'll change it.

Reply:
And I see now that "chaîne" should as well be in the plural form here in "chaînes de caractères" and should have a '^' accent on the 'i'.

Reply:
```suggestion
> Recherche les répertoires les plus utilisés et permet une navigation rapide à l'aide de chaînes de caractères ou d'expressions régulières.
```